### ISSUE TITLE
Scripts in html pane does not affect on js pane.

### ISSUE THREAD

**Issue by lapanoid** [ghinda/jotted#3]
Thanks for your project, it really great in terms of interability and api. I am building react component on top of it and it was a joy to hacking around it.

Currently I have weird and unexpected behavior when I run this:
```
{
	files: [{
      type: 'html',
      content: '<html>\n' +
				'	<head>\n' +
				'		<script src="https://fb.me/react-0.14.6.js"></script>\n' +
				'		<script src="https://fb.me/react-dom-0.14.6.js"></script>\n' +
				'	</head>\n' +
				'	<body>\n' +
				'		<div id="content"/>\n' +
				'	</body>\n' +
				'</html>\n'
    }, {
    	type: 'js',
    	content: 'ReactDOM.render(\n' +
				'	        <span>Hello, world!</span>,\n' +
				'	        document.getElementById("content")\n' +
				'	      );'
    }],
    plugins: [{
      name: 'babel',
      options: {
        presets: ['react']
      }
    }]
}
```

with result `ReactDOM is not defined`

There is another one to show problem closely:
```
{
	files: [{
      type: 'html',
      content: '<html>\n' +
				'	<head>\n' +
				'		<script src="https://fb.me/react-0.14.6.js"></script>\n' +
				'	</head>\n' +
				'</html>\n'
    }, {
      type: 'js',
      content: 'console.log(<div/>);'
    }],
    plugins: [{
      name: 'babel',
      options: {
        presets: ['react']
      }
    }]
}
```
with result `React is not defined`

Apparently js is running in isolated context, but not in html iframe. I was thinking to make PR for this, but not quite get code yet. Would be great if you help me here a bit.
Cheers.

**Comment by lapanoid** [comment]
It seems like it only happens on initial load, so we need to wait html scripts will be finished and then run js pane

**Comment by ghinda** [comment]
Fixed in `1.0.4`.

Scripts in the JavaScript pane will now run only after all inline scripts in the HTML pane are loaded.
After you upgrade, everything should be working.

Make sure you include babel's [browser.js](https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.34/browser.js) on the page.

Don't use the `presets` option, since it doesn't seem to be supported by `browser.js` 5.x, and throws an error.

So using something like this should work:

```
{
    files: [{
      type: 'html',
      content: '<script src="https://fb.me/react-0.14.6.js"></script>'
    }, {
      type: 'js',
      content: 'console.log(<div/>);'
    }],
    plugins: ['babel']
}
```

One more thing, you don't need to use `<head>` or `<body>` in the HTML content. All the content will be inserted in the iframe's body. Jotted creates the standard structure with a doctype and everything, itself.

Thanks again for reporting and testing! :beers:

**Comment by lapanoid** [comment]
@ghinda awesome! :beers:
